CMakeLists.txt

```diff
@@ -203,22 +207,25 @@ add_custom_target(combined_lib
 
 # Tests
 set(TEST_EXECS "")
-file(GLOB TEST_SRCS test/cpp/*.cc)
+file(GLOB TEST_SRCS tests/cpp/*.cc)
 find_library(GTEST_LIB NAMES libgtest.a gtest)
 
-if(GTEST_LIB)
```

Comment:
Why do we remove gtest? I think we need to keep them. And I think we need to tests for DLR using gtests.

Reply:
This c test is supposed to be run on both linux and android, using gtest adds an additional dependency.